docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+version: '3'
```

Comment:
Use the latest and greatest spec version which is 3.9 as far as I know

Reply:
Done